eng/Tools.props

```diff
@@ -11,6 +11,9 @@
 
     <!-- Need to keep in sync with CodeAnalysis.targets file. -->
     <AnalyzerPropsFile>$(ArtifactsToolsetDir)Common\Tools.Analyzers.props</AnalyzerPropsFile>
+
+    <!-- Restore specific version of NetStandard.library -->
+    <NetStandardImplicitPackageVersion>$(NetStandardLibraryPackageVersion)</NetStandardImplicitPackageVersion>
```

Comment:
Did you need to set this?  I would have expected that we would have disabled all the implicit package references

Reply:
The SDK packages all implicitly reference NetStandard.Library/Microsoft.NETCore.App, if I don't set this we'll restore the default version & I won't have the path to find that package in my package cache. If I set it here, this project will restore the version that I want so I know where to find it.

Reply:
Did you consider DisableImplicitFrameworkReferences?

Reply:
Actually I see we already use that in some projects.

Reply:
I see, are you sure we'll restore one of those projects that references it?  The other places where we're working around missing assets file make me think we may not be.